### PR TITLE
spaced out the buttons on main roster list to be responsive to smaller window

### PR DESCRIPTION
**Pull request by rayykim**
## Summary
- positioned the buttons

## Screenshots 

Before Update
screen shot 2016-08-16 at 11 24 29 am

After Update
screen shot 2016-08-16 at 1 12 50 pm

### CODE REVIEW

File: kolibri/plugins/management/assets/src/vue/user-page/index.vue [hunk 1]
```diff
@@ -277,16 +277,21 @@
...
       box-sizing: border-box
       width: 50%
+    #type-filter
+      position: relative
+      top: 5px
     .searchbar
-      font-size: 0.8em
-      width: 100%
+      font-size: 0.9em
+      width: 102%
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/vue/user-page/index.vue]: 102%? is this a typo?

File: kolibri/plugins/management/assets/src/vue/user-page/index.vue [hunk 2]
```diff
@@ -10,12 +10,12 @@
...
-        <option selected value="all"> All Users </option>
-        <option value="admin"> Admins </option>
-        <option value="learner"> Learners </option>
-      </select>
+        <label for="type-filter" class="visuallyhidden">Filter User Type</label>
+        <select v-model="roleFilter" id="type-filter" name="type-filter">
+          <option selected value="all"> All Users </option>
+          <option value="admin"> Admins </option>
+          <option value="learner"> Learners </option>
+        </select>
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/vue/user-page/index.vue]: looks like you may have accidentally changed the indentation?

File: kolibri/core/assets/src/vue/icon-button.vue [hunk 3]
```diff
@@ -72,6 +72,10 @@
     padding-right: 8px
     height: 36px
 
+  @media screen and (max-width: $portrait-breakpoint)
+    .icon-button
+      margin: 5px
+
```
Comment (indirectlylit) [inline kolibri/core/assets/src/vue/icon-button.vue]: it would be preferable to do this without setting a media query - is it possible to just always give it a 5px margin?

File: kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue [hunk 4]
```diff
@@ -144,6 +144,7 @@
       height: 40px
       font-weight: bold
       background-color: transparent
+      cursor: pointer
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue]: doesn't seem related to this PR

File: kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue [hunk 5]
```diff
@@ -223,6 +223,7 @@
       height: 40px
       font-weight: bold
       background-color: transparent
+      cursor: pointer
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue]: doesn't seem related to this PR

File: kolibri/core/assets/src/vue/icon-button.vue (hunk 3 above)
Comment (indirectlylit) [inline kolibri/core/assets/src/vue/icon-button.vue]: also note that this button is re-used in many places, so adding the style here potentially breaks those. the style should really be moved out

**Comment by rayykim** [comment]
@indirectlylit @DXCanas applied recommended changes

**Comment by DXCanas** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;9HPigt3G&#x2F;536-updates-to-user-management-page"> updates to user management page</a>
